### ISSUE TITLE
[JENKINS-70209] Letter spacing should not be negative (regression in 2.340 + 2.350)

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-23#7069]
<p>Negative letter spacing should be avoided. It greatly decreases legibility and accessibility.</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/59618/59618_image-2022-12-05-16-54-00-900.png" style="border: 0px solid black" /></span></p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gadzy">gadzy</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-70209" target="_blank">Letter spacing should not be negative (regression in 2.340 + 2.350)</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gadzy">gadzy</a>
<li><b>status</b>: Closed
<li><b>priority</b>: Minor
<li><b>component(s)</b>: core
<li><b>label(s)</b>: 2.375.3-fixed, regression, ux
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2023-01-01T19:49:12+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-23
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>Negative letter spacing should be avoided. It greatly decreases legibility and accessibility.</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/59618/59618_image-2022-12-05-16-54-00-900.png" style="border: 0px solid black" /></span></p></pre>
</details>
</details>
<details><summary><i>1 attachment</i></summary>

- [image-2022-12-05-16-54-00-900.png](https://issues.jenkins.io/secure/attachment/59618/image-2022-12-05-16-54-00-900.png)
  > ![image-2022-12-05-16-54-00-900.png](https://issues.jenkins.io/secure/attachment/59618/image-2022-12-05-16-54-00-900.png)
</details>
